xml/System.Diagnostics.Tracing/PollingCounter.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>PollingCounter is a variant of EventCounter - it collects and calculates similar statistics  as EventCounter. PollingCounter differs from EventCounter in that it takes in a callback function to collect metrics on its own rather than the user having to call WriteMetric() every time.</summary>
```

Comment:
```suggestion
    <summary><see cref="T:System.Diagnostics.Tracing.PollingCounter" /> is a variant of <see cref="T:System.Diagnostics.Tracing.EventCounter" /> - it collects and calculates similar statistics  as <see cref="T:System.Diagnostics.Tracing.EventCounter" />.
    
    <see cref="T:System.Diagnostics.Tracing.PollingCounter" /> differs from <see cref="T:System.Diagnostics.Tracing.EventCounter" /> in that it takes in a callback function to collect metrics on its own rather than the user having to call <see cref="M:System.Diagnostics.Tracing.EventCounter.WriteMetric(System.Double)" /> every time.</summary>
```